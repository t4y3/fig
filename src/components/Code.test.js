import { h } from 'hyperapp';
import { renderToString } from 'hyperapp-render';
import hljs from 'highlight.js';
import Code from './Code';

global.hljs = hljs;

test('Code.js to match snapshot', () => {
  const view = renderToString(<Code template="<p>test</p>" />);
  expect(view).toMatchSnapshot();
});
