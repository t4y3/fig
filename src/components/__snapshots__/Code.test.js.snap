// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Code.js to match snapshot 1`] = `"<div className=\\"fig-code\\"><div className=\\"siimple-h5\\">Source</div><pre className=\\"siimple-pre\\"><code className=\\"html\\"><span class=\\"hljs-tag\\">&lt;<span class=\\"hljs-name\\">p</span>&gt;</span>test<span class=\\"hljs-tag\\">&lt;/<span class=\\"hljs-name\\">p</span>&gt;</span></code></pre></div>"`;
